crates/rome_js_formatter/src/js/expressions/unary_expression.rs

```diff
@@ -27,10 +29,18 @@ impl FormatNode for JsUnaryExpression {
         );
 
         if is_keyword_operator {
+            let needs_parenthesis = JsAwaitExpression::can_cast(argument.syntax().kind());
```

Comment:
nit: we can do a match on argument directly: 
```suggestion
            let needs_parenthesis = matches!(argument, JsAnyExpression::JsAwaitExpression(_));
```